sharding/data_modeling.rst

```diff
@@ -1,7 +1,7 @@
-.. _distributed_tables:
+.. _distributed_data_modeling:
 
 Picking a Distribution Column
-#############################
+-----------------------------
 
 Every distributed table in Citus has exactly one column which is chosen as the distribution column. This informs the database to maintain statistics about the distribution column in each shard. Citus’s distributed query optimizer then leverages these statistics to determine how best a query should be executed.
```

Comment:
This paragraph is OK to remain, but the below two seem like they should be maybe after the use-case discussion?